Description: Enable all cgroup controllers in logind

This changes the default logind configuration to enable all cgroup
controllers. Doing so will result in a /user/$UID.user/cX.session cgroup
being created for each of the listed controller with the cX.session
entry belonging to the user.

In the event that one of the listed controllers doesn't exist under
/sys/fs/cgroup, logind will simply ignore it.
The list in this patch should be kept up to date as new cgroup
controllers are added to the kernel and the distro (the config option
unfortunately doesn't support wildcards).

This configuration is required, almongst other things by LXC for
unprivileged containers.

Origin: vendor
Forwarded: not-needed

Index: systemd-204/src/login/logind.conf
===================================================================
--- systemd-204.orig/src/login/logind.conf	2014-05-08 18:30:35.349061594 -0500
+++ systemd-204/src/login/logind.conf	2014-05-08 18:30:52.377060908 -0500
@@ -13,8 +13,8 @@
 #KillUserProcesses=no
 #KillOnlyUsers=
 #KillExcludeUsers=root
-#Controllers=
-#ResetControllers=cpu
+Controllers=blkio cpu cpuacct cpuset devices freezer hugetlb memory perf_event net_cls net_prio
+ResetControllers=
 #InhibitDelayMaxSec=5
 #HandlePowerKey=poweroff
 #HandleSuspendKey=suspend
